tools/generate_uuids.py

```diff
@@ -4,10 +4,12 @@
 
 parser = argparse.ArgumentParser(description="Generate UUIDs for info*.json files that don't already have them.")
 parser.add_argument("directory", help="the directory to search for info*.json files")
+parser.add_argument("-v", "--verbose", action="store_true", help="List all the files for which uuid's were generated")
```

Comment:
UUID should be capitalized.

No apostrophe in `UUIDs` (it's a plural, not a possessive).

Argument help should start with a lowercase character.